src/locales/ko/mystery-encounters/slumbering-snorlax-dialogue.json

```
@@ -1,25 +1,25 @@
 {
-  "intro": "As you walk down a narrow pathway, you see a towering silhouette blocking your path.$You get closer to see a {{snorlaxName}} sleeping peacefully.\nIt seems like there's no way around it.",
-  "title": "Slumbering {{snorlaxName}}",
-  "description": "You could attack it to try and get it to move, or simply wait for it to wake up. Who knows how long that could take, though...",
-  "query": "What will you do?",
+  "intro": "좁은 길을 걷다보니, 우뚝 솟은 거대한 실루엣이 길을 막고 있는 것을 발견했다.$가까이 다가가보니, 평화롭게 자고있는 {{snorlaxName}}이다.\n피할 수 있는 방법은 없는 것 같다….",
+  "title": "잠자는 {{snorlaxName}}",
+  "description": "공격해서 자리를 비키게 할 수도, 잠에서 깨어날 때까지 기다릴 수도 있습니다. 하지만 얼마나 오래 걸릴지는 알 수 없겠지만요….",
```

```suggestion
  "description": "공격해서 자리를 비키게 할 수도, 잠에서 깨어날 때까지 기다릴 수도 있습니다. 얼마나 기다려야 할지는 알 수 없겠지만요….",
```
그냥 '걸린다'라고만 하면 앞(공격하는 것)에도 해당이 되는 것처럼 읽힐 수도 있을 듯하여 '기다리다'로 맞추어 주는 게 더 이해하기 쉬울 듯합니다.